effekt/jvm/src/test/scala/effekt/LLVMTests.scala

```diff
@@ -39,6 +39,8 @@ class LLVMTests extends EffektTests {
     examplesDir / "benchmarks" / "are_we_fast_yet" / "queens.effekt",
     examplesDir / "benchmarks" / "input_output" / "word_count_ascii.effekt",
     examplesDir / "benchmarks" / "input_output" / "word_count_utf8.effekt",
+    examplesDir / "benchmarks" / "input_output" / "dyck_one.effekt",
+    examplesDir / "benchmarks" / "input_output" / "number_matrix.effekt",
```

Comment:
This one should really **not** be ignored straight away!
Valgrind complains about something freed too early when reading a file: cc @marvinborner 
```
-[error] Process exited with non-zero exit code 139.
-[error] Valgrind log:
-==11975== Invalid read of size 8
-==11975==    at 0x10FE9B: returnAddress_1207 (in /home/runner/work/effekt/effekt/out/tests/effekt.llvmtests/number_matrix)
-==11975==    by 0x10B598: resume_Int (in /home/runner/work/effekt/effekt/out/tests/effekt.llvmtests/number_matrix)
-==11975==    by 0x109EB5: c_resume_int_fs (io.c:62)
```